services/ros_flask_server/server.py

```diff
@@ -0,0 +1,137 @@
+import threading
+import rospy
+
+from std_msgs.msg import String
+from flask import Flask, request
+from flask import jsonify
+
+import logging
+import time
+from os import getenv
+
+import sentry_sdk
+
+
+sentry_sdk.init(getenv("SENTRY_DSN"))
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+
+talker = rospy.Publisher("talker", String, queue_size=1)
+
+threading.Thread(target=lambda: rospy.init_node("listener", disable_signals=True)).start()
+
+VALID_COMMANDS = ["test_command"]
+COMMAND_QUEUE = []
+EXECUTING_COMMAND = None
```

Comment:
it does not have endpoint ping here. is it correct? ping is required for is_container_running function

Reply:
is_container_running worked correctly (or it seemed like it) without it. endpoint added